pyraphtory/Cargo.toml

```diff
@@ -9,7 +9,7 @@ name = "pyraphtory"
 crate-type = ["cdylib"]
 
 [dependencies]
-pyo3 = { version = "0.17.1", features = ["extension-module"] }
+pyo3 = { version = "0.17.1" } #, features = ["extension-module"] }
```

Comment:
@Haaroon I don't think this is supposed to be removed? probably accidentally committed

Reply:
It broke again for me :( when I removed it was fine so bye bye

Reply:
We need to upgrade to `0.18.1` and use the virtual-env with Python 3.10